tests/test_text.py

```diff
@@ -0,0 +1,9 @@
+from manim import Text
+
+
+def test_long_string():
+    """Check that `#469 <https://github.com/ManimCommunity/manim/issues/469>`_
+    is resolved."""
+
+    t = Text("a" * 150, font="Arial")
+    assert len(t.text) == len(t.submobjects)
```

Comment:
Pretty much this won't work at all. Because I had found it out while working on #471 that Cairo will produce some empty `path` in SVG. Manim just ignores those empty paths. So checking this won't work.
https://github.com/ManimCommunity/manim/blob/650d1528fbabf9e326433e7e702e6233c6f9afeb/manim/mobject/svg/svg_mobject.py#L90-L93

Reply:
The fact that the text length does not coincide with the number of submobjects is precisely what causes the error in #469, you can check that we have this equality for `Text("a" * 100)`, but not for `Text("a" * 150)`.

Reply:
Pretty much I tried a lot on that. and possibly removing
https://github.com/ManimCommunity/manim/blob/master/manim/mobject/svg/text_mobject.py#L152-L173
worked out. With some of its references.

Reply:
That worked out for Pango but I don't think it will work here sadly.

Reply:
I am not sure what you are saying. What do you mean by "removing" `gen_chars`?

Just to be clear: the changes proposed here simply guarantee that the entire text will fit onto the canvas the svg is rendered on, ensuring that arbitrarily long strings can be handled. (The canvas actually gets way to large by this approach, but I could not find a significant impact to render speed.)

In particular, these changes allow the string given in #469 and the test string `Text("a" * 120)` to be rendered.

When you say that this approach will not work, do you have a concrete example of a text that fails to render here?

Reply:
Pretty much on `Text("a" * 150)` text is still half-written by Cairo sadly.
![image](https://user-images.githubusercontent.com/49693820/94249133-147c4d80-ff3d-11ea-9998-02dd67a9742d.png)

Reply:
Where is this image from? When I compile the scene in this example:

```py
class Test(Scene):
    def construct(self):
        t = "a" * 150 + "b"
        text = Text(t, font='Arial', stroke_width=0)

        self.add(text.scale(0.3))
        self.wait(5)
```

I get

![image](https://user-images.githubusercontent.com/11851593/94250011-f143a500-ff20-11ea-9187-cbc78d3e580c.png)

The string is rendered and displayed in full here.

Reply:
That image is from my browser when I opened the svg file.

Reply:
Same script for me 
```py
class Test(Scene):
    def construct(self):
        t = "a" * 150 + "b"
        text = Text(t, font='Arial', stroke_width=0)

        self.add(text.scale(0.3))
        self.wait(5)
```
![image](https://user-images.githubusercontent.com/49693820/94254941-1e09b380-ff45-11ea-8f96-0c6117bb2959.png)

Reply:
I am clueless, I don't really see a reason for the different behavior you encounter. Did you compile this with an empty `media/texts` directory? (Just to exclude caching issues.)

The only significant difference is probably that you are using manim from Windows, right? (I've tried this fix on three different linux machines, I always get the full string rendered.)

Reply:
Pretty much, yes, I was using the master branch. (Forgot to run poetry install). It works now.
![image](https://user-images.githubusercontent.com/49693820/94263144-3af8b380-ff52-11ea-90b9-7c84e9629c92.png)